internal/pkg/featureflags/disable.go

```diff
@@ -0,0 +1,54 @@
+package featureflags
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	dynamicconfig "github.com/confluentinc/cli/internal/pkg/dynamic-config"
+	ppanic "github.com/confluentinc/cli/internal/pkg/panic-recovery"
+)
+
+func DisableHelpText(command *cobra.Command, flags []string) {
+	if len(flags) == 0 {
+		command.Hidden = true
+	} else {
+		formattedFlags := ppanic.ParseFlags(command, flags)
+		for _, flag := range formattedFlags {
+			err := command.Flags().MarkHidden(flag)
+			if err != nil {
+				return
+			}
```

Comment:
What if one flag doesn't exist but another does?
```suggestion
			_ = command.Flags().MarkHidden(flag)
```